packages/space-opera/src/components/model_viewer_preview/model_viewer_preview.ts

```diff
@@ -250,9 +250,12 @@ export class ModelViewerPreview extends ConnectedLitElement {
             childElements)}`;
   }
 
+  // Handle the case when the model is loaded for the first time.
   private onModelLoaded() {
-    // Handle the case when the model is loaded for the first time.
-    this.onGltfUrlChanged();
+    // only update on poster reveal
+    if (this.modelViewer && this.modelViewer.reveal === 'interaction') {
+      this.onGltfUrlChanged();
```

Comment:
Can we replace this logic by using `showPoster()` now?

Reply:
yeah, in the next PR I was just going to completely update the poster logic, i.e. also removing the difference between create/display poster